Update internal.md (#1407)
gohugoio/hugoDocs: content/en/templates/internal.md

Hugo uses the page title and description for the title and description metadata.
The first 6 URLs from the `images` array are used for image metadata.

> Furthermore, if [page bundles](/content-management/page-bundles/) are used and no `images` array is defined, images with filenames containing `*feature*` or `*cover*,*thumbnail*` are automatically used for image metadata.

Reviewer: ```suggestion
If [page bundles](/content-management/page-bundles/) are used and the `images` array is empty or undefined, images with filenames matching `*feature*` or `*cover*,*thumbnail*` are used for image metadata.
```
